onnx/defs/math/defs.cc

```diff
@@ -2847,6 +2853,13 @@ bool BuildContextDependentFunctionBody(
     const FunctionBodyBuildContext& ctx,
     const OpSchema& schema,
     FunctionProto& functionProto) {
+  bool float_input = true;
+  if (ctx.getInputType(0) != nullptr) {
+    float_input = ctx.getInputType(0)->tensor_type().elem_type() == TensorProto_DataType_FLOAT;
```

Comment:
minor nit: ```ctx.getInputType(0)->tensor_type().elem_type()``` is repeated thrice. May be worth making it a variable. Changing the control-flow to ```if (ctx.getInputType(0) == nullptr) return;``` would make it doable.